frontend/src/views/assignments/withdraw-assignment-confirmation.js

```diff
@@ -0,0 +1,101 @@
+import { useDispatch } from "react-redux";
+import React from "react";
+import { AdvancedFilterTable } from "../../components/filter-table/advanced-filter-table";
+import { Button, Modal } from "react-bootstrap";
+
+function compareString(str1, str2) {
+    if (str1 > str2) {
+        return 1;
+    } else if (str1 < str2) {
+        return -1;
+    }
+    return 0;
+}
+
+export function MultiWithdrawOfferConfirmation(props) {
+    const dispatch = useDispatch();
+    const { data, visible, setVisible, withdrawOffers } = props;
+
+    // We want to minimize the re-render of the table. Since some bindings for columns
+    // are generated on-the-fly, memoize the result so we don't trigger unneeded re-renders.
+
+    data.sort((a1, a2) => {
```

Comment:
You should make a separate `compareAssignment` function.

Reply:
I wonder if we really need to sort them in advance? Because the table component we are using already has a sorting feature.

Reply:
> using

Yes we do. Last meeting we discussed that they need to be sorted by this order by default, but still support additional sorting.